src/player/health.js

```diff
@@ -46,18 +52,62 @@ export default {
       })
     })
 
+    aiter(abortable(setInterval(1000, null, { signal })))
+      .map(get_state)
+      .filter(state => !!state)
```

Comment:
Nit: to be consistent with another part of the code

```suggestion
      ..dropWhile(state => !state)
```